calico_containers/tests/unit/node_test.py

```diff
@@ -166,6 +166,98 @@ def test_validate_arguments(self, case, sys_exit_called):
     @patch('calico_ctl.node.docker', autospec=True)
     @patch('calico_ctl.node._find_or_pull_node_image', autospec=True)
     @patch('calico_ctl.node._attach_and_stream', autospec=True)
+    def test_node_dockerless_start(self, m_attach_and_stream,
+                                   m_find_or_pull_node_image, m_docker,
+                                   m_docker_client, m_client, m_install_plugin,
+                                   m_warn_if_hostname_conflict, m_warn_if_unknown_ip,
+                                   m_get_host_ips, m_setup_ip, m_check_system,
+                                   m_os_makedirs, m_os_path_exists):
+        """
+        Test that the node_start function behaves as expected when
```

Comment:
"behaves as expected" is pretty meaningless in this context.  How about "node_start function does not make Docker calls when runtime=None"